src/browser/ui/ReactMount.js

```diff
@@ -451,12 +457,13 @@ var ReactMount = {
    * perform an update on it and only mutate the DOM as necessary to reflect the
    * latest React component.
    *
+   * @param {ReactElement} parentComponent The conceptual parent of this render tree.
    * @param {ReactElement} nextElement Component element to render.
    * @param {DOMElement} container DOM element to render into.
    * @param {?function} callback function triggered on completion
    * @return {ReactComponent} Component instance rendered in `container`.
    */
-  render: function(nextElement, container, callback) {
+  renderSubtreeIntoContainer: function(parentComponent, nextElement, container, callback) {
```

Comment:
You didn't address the first part of this. What about warnings when called directly without a `parentComponent`?